_config.yml

```diff
@@ -1,39 +1,32 @@
 title: Marks and Spencer Engineering
 description: Digital Garden for Ideas, Voice and an Open culture 
 email: softwareengineering@marksandspencer.com
-
-# Do not use the leading @ here
 twitter: marksandspencerTech
 
-url:
+url: "https://marksandspencer.com"
```

Comment:
Should this be the blog url?

Reply:
Probably actually.  I kept the one that was here (there were two `url:`s).  Let me find out what it does.